charts/castai-sec-agent/values.yaml

```diff
@@ -0,0 +1,102 @@
+# Default values for castai-sec-agent.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicas: 2
+leaderElectionEnabled: true
+
+nameOverride: ""
+fullnameOverride: castai-sec-agent
+
+# By default namespace is expected to be created by castai-agent.
+createNamespace: false
+
+image:
+  repository: us-docker.pkg.dev/castai-hub/library/sec-agent
+  # Tag is set using Chart.yaml appVersion field.
+  tag: ""
+  pullPolicy: IfNotPresent
+
+imagePullSecrets: {}
+
+# Controls `deployment.spec.strategy` field
+updateStrategy:
+  type: Recreate
+
+# CAST AI API configuration.
+castai:
+  # Token to be used for authorizing agent access to the CASTAI API
+  apiKey: ""
+
+  # Name of secret with Token to be used for authorizing agent access to the API
+  # apiKey and apiKeySecretRef are mutually exclusive
+  # The referenced secret must provide the token in .data["API_KEY"]
+  apiKeySecretRef: ""
+
+  # CASTAI public api url.
+  apiURL: "https://api.cast.ai"
+
+  # CASTAI Cluster unique identifier.
+  clusterID: ""
+
+podAnnotations: {}
+podLabels: {}
+
+additionalEnv:
+  LOG_LEVEL: "5"
+
+resources:
+  requests:
+    cpu: 50m
+    memory: 100Mi
+  limits:
+    memory: 1Gi
+
+# Pod toleration rules.
+# Ref: https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/
+tolerations: {}
+
+# Pod node selector rules.
+# Ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/
+nodeSelector: {}
+
+# Pod affinity rules.
+# Ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity
+affinity:
+  nodeAffinity:
+    requiredDuringSchedulingIgnoredDuringExecution:
+      nodeSelectorTerms:
+        - matchExpressions:
+            - key: kubernetes.io/os
+              operator: NotIn
+              values:
+                - windows
+  podAntiAffinity:
+    preferredDuringSchedulingIgnoredDuringExecution:
+      - weight: 100
+        podAffinityTerm:
+          labelSelector:
+            matchExpressions:
+              - key: app.kubernetes.io/name
+                operator: In
+                values:
+                  - castai-sec-agent
+          topologyKey: kubernetes.io/hostname
+
+# Pod security context.
+# Ref: https://kubernetes.io/docs/tasks/configure-pod-container/security-context/
+securityContext: {}
+
+serviceAccount:
+  create: true
+  # Annotations to add to the service account.
+  annotations: {}
+  name: "castai-sec-agent"
+
+# Settings for configuring k8s client used in castai-sec-agent.
+k8sApiClient:
```

Comment:
I think we won't need this (for now)

Reply:
this is for ratelimiting requests to kube api. Im sure we will need this when we will start creating containers with our tools.